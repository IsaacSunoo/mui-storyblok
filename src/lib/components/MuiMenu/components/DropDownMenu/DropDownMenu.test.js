import React from 'react';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import DropDownMenu from './DropDownMenu';

function setup() {
  const props = {};
  const comp = shallow(
    <DropDownMenu {...props}>
      <p>here</p>
    </DropDownMenu>,
  );
  return { comp, props };
}

describe('<DropDownMenu />', () => {
  it('renders DropDownMenu', () => {
    const { comp } = setup();
    expect(comp).toBeDefined();
  });

  test('snapshot with auth', () => {
    const { props } = setup();

    const tree = renderer.create(
      <MemoryRouter>
        <DropDownMenu {...props}>
          <p>here</p>
        </DropDownMenu>
      </MemoryRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
