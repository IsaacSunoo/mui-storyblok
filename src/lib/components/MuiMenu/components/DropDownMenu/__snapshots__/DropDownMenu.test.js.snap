// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropDownMenu /> snapshot with auth 1`] = `
<div
  className={undefined}
>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textSecondary"
    disabled={false}
    onBlur={[Function]}
    onClick={undefined}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      
      <span
        className="MuiButton-endIcon MuiButton-iconSizeMedium"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root"
          color={undefined}
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </span>
    </span>
  </button>
  <div
    className="MuiPaper-root MuiPaper-outlined MuiPaper-rounded"
  >
    <p>
      here
    </p>
  </div>
</div>
`;
